src/Components/Endpoints/src/RazorComponentEndpointInvoker.cs

```diff
@@ -145,8 +149,17 @@ private async Task<RequestValidationState> ValidateRequestAsync(IAntiforgery? an
     private string? GetFormHandler(out bool isBadRequest)
     {
         isBadRequest = false;
+        var hasValue = false;
+        StringValues value = StringValues.Empty;
+        try
+        {
+            hasValue = _context.Request.Form.TryGetValue("_handler", out value);
```

Comment:
Means we're going to make sure we do a performance run.

Reply:
Yes, I have a note to measure the impact of the additional check in the `EndpointMiddleware`.

cc: @javiercn for the note on RazorComponentEndpointInvoker

Reply:
The code has already run antiforgery validation before we get to this line, so doesn't that guarantee it's already read the form data anyway? Or does antiforgery validation somehow not involve doing that? Or is this only a concern in the case where antiforgery validation is disabled?

Either way I certainly see it would be clearer if we used `ReadFormAsync` here. Just checking if my understanding is correct.